docs/image-operations.md

```diff
@@ -0,0 +1,147 @@
+![Cereal resized only one dimention](../images/cereal-ops/cereal-resized-width.jpeg)
+
+When running inferences, resizing images can help reduce the traffic and achieve faster inferences. But don't downscale it too much, or you might lose important image details.
+
+## Color
+
+You can adjust color intensity. The factor parameter must be 1.0 for no change, less than 1.0 for less intensity and greater than 1.0 for more intensity.
+
+```python
+from landingai.pipeline.frameset import Frame
+
+# Adjust color to 0.1
+frame = Frame.from_image("docs/images/cereal-ops/cereal.jpeg")
+frame.adjust_color(0.1)
+frame.save_image("docs/images/cereal-ops/cereal-color-0.1.jpeg")
+
+# Adjust color to 2.0
+frame = Frame.from_image("docs/images/cereal-ops/cereal.jpeg")
+frame.adjust_color(2.0)
+frame.save_image("docs/images/cereal-ops/cereal-color-2.0.jpeg")
+```
+
+![Cereal image](../images/cereal-ops/cereal.jpeg)
+![Cereal low color adjustment](../images/cereal-ops/cereal-color-0.1.jpeg)
+![Cereal high color adjustment](../images/cereal-ops/cereal-color-2.0.jpeg)
+
+## Contrast
+
+Similar to color, you can adjust the contrast. The factor parameter must be 1.0 for no change, less than 1.0 for less intensity and greater than 1.0 for more intensity.
+
+```python
+from landingai.pipeline.frameset import Frame
+
+# Adjust contrast to 0.1
+frame = Frame.from_image("docs/images/cereal-ops/cereal.jpeg")
+frame.adjust_contrast(0.1)
+frame.save_image("docs/images/cereal-ops/cereal-contrast-0.1.jpeg")
+
+# Adjust contrast to 2.0
+frame = Frame.from_image("docs/images/cereal-ops/cereal.jpeg")
+frame.adjust_contrast(2.0)
+frame.save_image("docs/images/cereal-ops/cereal-contrast-2.0.jpeg")
+```
+
+
+![Cereal image](../images/cereal-ops/cereal.jpeg)
+![Cereal low contrast adjustment](../images/cereal-ops/cereal-contrast-0.1.jpeg)
+![Cereal high contrast adjustment](../images/cereal-ops/cereal-contrast-2.0.jpeg)
+
+## Brightness
+
+To adjust brightness, use the same factor parameter: 1.0 for no change, less than 1.0 for less intensity and greater than 1.0 for more intensity.
+
+```python
+from landingai.pipeline.frameset import Frame
+
+# Adjust brightness to 0.1
+frame = Frame.from_image("docs/images/cereal-ops/cereal.jpeg")
+frame.adjust_brightness(0.1)
+frame.save_image("docs/images/cereal-ops/cereal-brightness-0.1.jpeg")
+
+# Adjust brightness to 2.0
+frame = Frame.from_image("docs/images/cereal-ops/cereal.jpeg")
+frame.adjust_brightness(2.0)
+frame.save_image("docs/images/cereal-ops/cereal-brightness-2.0.jpeg")
+```
+
+
+![Cereal image](../images/cereal-ops/cereal.jpeg)
+![Cereal low brightness adjustment](../images/cereal-ops/cereal-brightness-0.1.jpeg)
+![Cereal high brightness adjustment](../images/cereal-ops/cereal-brightness-2.0.jpeg)
+
+## Sharpness
+
+Adjusting sharpness follows the same pattern as color, brightness and contrast: 1.0 for no change, less than 1.0 for less intensity and greater than 1.0 for more intensity.
```

Comment:
Reworded:
You can adjust the sharpness intensity:

- No change: 1.0
- Less intensity: Less than 1.0
- More intensity: Greater than 1.0

See the following examples.